data/hails.txt

```diff
@@ -465,10 +488,12 @@ phrase "friendly civilian"
 		"the best of luck"
 		"fair winds and bright skies"
 		"Allah"
+		"Huitzilopochtli"
 		"the power of Heaven"
 		"Heaven's blessings"
 		"the Way and the Light"
 		"the immortal Tao"
+		"Kek, the great"
```

Comment:
Still disapprove. Contemporary internet memes don't belong in this hail IMHO.

Reply:
dont care

Reply:
if you don't care about having your additions fit the game quality and you just want some stupid memes in it despite people saying it doesn't "belong", I think your attitude could use an upgrade

Reply:
Allah is there, which is arguably the most dangerous god that humans have created, focused on the harm of innocents with the pedophile prophet Mohammad. if this is allowed, why not kek who is there to make you laugh?

Reply:
ehm... what exactly are you talking about? nothing of that is in the game itself, and that comment is quite inappropriate
"Kek" is allowed to exist in general, but we'd prefer to keep the game... meme-free so to say

Reply:
The fourth wall is enjoyable to break, so long as they're uncommon and normally subtle. Like mentioning the helix, when referring to the helix fossil. I agree they shouldn't be super common.

Reply:
>Allah is there, which is arguably the most dangerous god that humans have created, focused on the harm of innocents with the pedophile prophet Mohammad. if this is allowed, why not kek who is there to make you laugh?

Lmfao, hide your power level, man.

Honestly, I see no harm in the inclusion of Kek. We're already making an assumption that any modern religions will survive through hundreds of years of space flight. Harmony is even a world of Amish people. Heck, we aren't even arguing about @greatkek's `Huitzilopochtli` addition, which from what I can tell has to do with some Aztec god. Also, Taosim (`"the imortal Tao"`) isn't a very widely practiced religion today (I found it to only have 173 million adherents), and from what I understand it won't be around (in name, but maybe still in spirit) for much longer, yet it's considered prominent enough to exist within a hail.

Reply:
> Allah is there, which is arguably the most dangerous god that humans have created, focused on the harm of innocents with the pedophile prophet Mohammad. if this is allowed, why not kek who is there to make you laugh?

Someone's starting on theThanksgiving dinner table conversations a bit early! :grimacing:

Reply:
In all honesty, what is a meme but a reference? There are plenty of references throughout the story. Are memes just too low-brow for you, @Pointedstick \:)

Including a small number of meme references in hails (which, to be honest, who reads every single hail they get anyways?) can't hurt. If anything, I think it'd be great to imagine someone playing the game, reading that hail, and laughing their ass off when they spot it, even if most people don't get it. Just like I'm sure most of the references in the story go over our heads too. That's kind of the whole point of references: only a few people will get them. It doesn't take anything away from people who don't get the references and it adds a lot for people who do.

It's all up to MZ though, so we'll just have to sit and wait.

Reply:
> Are memes just too low-brow for you, @Pointedstick :)

I like a good inside joke or reference as much as anyone (and have added many myself), but internet memes have some distinct disadvantages when it comes to semi-permanent content in a game:

1. They tend to be really juvenile since most of them are made by teenagers, and conflict with a more mature tone when such a thing is desired.
2. They go out of fashion quickly and tend not to hold up well. A meme will seem dated in mere months.